volatility3/framework/plugins/linux/psscan.py

```diff
@@ -0,0 +1,127 @@
+# This file is Copyright 2021 Volatility Foundation and licensed under the Volatility Software License 1.0
```

Comment:
Sorry, it's extremely minor, but please could you update the copyright year to the year it gets contributed during which you contribute it?